WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -159,12 +159,19 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
         updateGhostableTableViewOptions()
 
         configureNavigationButtons()
+
+        createButtonCoordinator.add(to: view, trailingAnchor: view.safeAreaLayoutGuide.trailingAnchor, bottomAnchor: view.safeAreaLayoutGuide.bottomAnchor)
     }
 
+    lazy var createButtonCoordinator: CreateButtonCoordinator = { return CreateButtonCoordinator(self, newPost: {}, newPage: {}, newStory: {})
```

Comment:
Maybe add a new line for `return ...`?

Reply:
Oh no! All of the changes in this file should be excluded from this PR. 🤦‍♂️